### PR TITLE
travis: PHP 7.0 nightly added

### PR DESCRIPTION
**Pull request by TomasVotruba**
Also fixed standard to 2 spaces, that's used above

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,33 +24,34 @@ install:
...
-        env: DB=pgsql POSTGRESQL_VERSION=9.2 # driver currently unsupported by HHVM
-      - php: hhvm
-        env: DB=pgsql POSTGRESQL_VERSION=9.3 # driver currently unsupported by HHVM
-      - php: hhvm-nightly
-        env: DB=pgsql POSTGRESQL_VERSION=9.1 # driver currently unsupported by HHVM
-      - php: hhvm-nightly
-        env: DB=pgsql POSTGRESQL_VERSION=9.2 # driver currently unsupported by HHVM
-      - php: hhvm-nightly
-        env: DB=pgsql POSTGRESQL_VERSION=9.3 # driver currently unsupported by HHVM
+    - php: 7.0
```
Comment (stof) [inline .travis.yml]: you are excluding all PHP 7 builds here again, meaning your change does not add them

Comment (TomasVotruba) [inline .travis.yml]: Oh, it's wrong section. I didn't notice. Thanks

Comment (TomasVotruba) [inline .travis.yml]: Fixed!

**Comment by stof** [comment]
:+1:

**Comment by Ocramius** [comment]
:+1: as well

**Comment by TomasVotruba** [comment]
:+1:

**Pull request merged**